org.jacoco.doc/docroot/doc/ant.html

```diff
@@ -857,11 +857,11 @@ <h3>Element <code>check</code></h3>
     </tr>
     <tr>
       <td><code>minimum</code></td>
-      <td>Expected minimum value. If the minimum refers to a ratio the range is
-          from 0.0 to 1.0 where the number of decimal places will also determine
-          the precision in error messages. A limit ratio may optionally be
-          declared as a percentage where 0.80 and 80% represent the same value,
-          the value must end with %.</td>
+      <td>Expected minimum value. If the minimum refers to a ratio it must be
+          in the range from 0.0 to 1.0 where the number of decimal places will
+	      also determine the precision in error messages. A limit ratio may
+	      optionally be declared as a percentage where 0.80 and 80% represent
+	      the same value.</td>
```

Comment:
This and other HTML files use spaces for indentation, so for consistency tab characters should be removed.